### PR TITLE
Tracker: Add starting inventory to trackers and received items table.

### PR DESCRIPTION
**Pull request by ThePhar**
## What is this fixing or adding?
Adds the starting inventory back into the inventory tables for trackers. Also adds those pre-collected items to received items on generic tracker.

## How was this tested?
Ran WebHost and checked with a SC2 game with starting items.

## If this makes graphical changes, please attach screenshots.
image
image

### CODE REVIEW

**Review by neocerber — APPROVED** [review]
The proposed changes does what the title state: 
- For specific tracker, it add the items that are in a player starting_inventory in the inventory used by the tracker. 
    - `get_player_inventory_counts()` is used explicitly, for most game, to define the inventory so it shouldn't have an impact on anything else. The only cases that it is not straightforward for me is Factorio and I am not sure why. However, it was already like that prior the proposed changes.
- For the genereric tracker, it add the items that are in a player starting_inventory in the dictionary used to list the item in "Last Order Received". The proposal takes into account that starting_inventory occur before any receiving, as it should!

Testing: Generated a multiworld of SC2, OoT, TimeSpinner and Factorio, each with starting items, some of them with counts > 1 and some progressive. Check both generic and specific tracker of each world; before and after collect .

Results: 
- Before collect: 
   - For all game, the generic tracker and the specific tracker showed the items in starting inventory, as intended!
- After collect:
    - For all game except SC2, the generic tracker and the specific tracker showed all the items and the "Last Order Received" seems to made sense. (Would need to send item one by one to be sure, not done.)
    - For SC2: Everything is find except for starting item that have a count > 1. In those cases, only the ones in the starting inventory are shown.
        - After investigation, I concluded that SC2 exclude all other instances of a Progressive Item if the said Progressive Item is in the starting inventory. The [issue](https://github.com/Ziktofel/Archipelago/issues/178) was reported in the world dev repo. 
        - If the previous issue is patched, its generic tracker and specific tracker work, as intended with the current proposal!

Comment: It is confusing for me to have the variable `item` being a structure and an Id (e.g. `item.item`). However, it is often used in Archipelago, so it seems to be intentional. 

TL;DR: Does what it claims, testing shows it works has intended.

**Review by Ziktofel — APPROVED** [review]

**Comment by Berserker66** [comment]
> 
> Comment: It is confusing for me to have the variable `item` being a structure and an Id (e.g. `item.item`). However, it is often used in Archipelago, so it seems to be intentional.
> 

technically, the pep8 compliant naming  would be network_item.item

**Review by Berserker66 — APPROVED** [review]
